manim/utils/space_ops.py

```diff
@@ -246,12 +249,12 @@ def angle_between_vectors(v1, v2):
         )
 
 
-def project_along_vector(point, vector):
+def project_along_vector(point: float, vector: np.ndarray) -> np.ndarray:
     matrix = np.identity(3) - np.outer(vector, vector)
     return np.dot(point, matrix.T)
 
 
-def normalize(vect, fall_back=None):
+def normalize(vect: np.ndarray, fall_back=None) -> np.ndarray:
```

Comment:
Sequence[float] ?

Reply:
No it should be np.ndarray as you would take the vector cords in an numpy array

Reply:
print(normalize(np.array([1,1,1])))
print(normalize([1,1,1]))
is both possible.
Why should it be limited to np.ndarrays?